packages/terra-clinical-header/src/Header.jsx

```diff
@@ -44,10 +47,7 @@ const defaultProps = {
 const Header = ({
   children, title, startContent, endContent, isSubheader, ...customProps
 }) => {
-  let startElement;
-  if (startContent) {
-    startElement = <div className={cx('flex-end')}>{startContent}</div>;
-  }
+  const theme = useContext(ThemeContext) || {};
```

Comment:
I'm falling back to `{}` for any components rendered in a project that does not have the context defined

Reply:
Can the context do that by default? I was thinking thats what this would do:
https://github.com/cerner/terra-framework/blob/master/packages/terra-theme-context/src/ThemeContext.jsx#L4

Reply:
OH I didn't realize there was a default. I'll assume it works fine then. 👍